content/help/ops/security/debugging-authorization/index.md

```diff
@@ -0,0 +1,257 @@
+    {{< /text >}}
+
+1. Verify you see the following output:
+
+    {{< text plain >}}
+    Forwarding from 127.0.0.1:9876 -> 9876
+    {{< /text >}}
+
+1. Start your browser and open the `ControlZ` page at `http://127.0.0.1:9876/scopez/`.
+
+1. Change the `rbac` Output Level to `debug`.
+
+1. Use `Ctrl+C` in the terminal you started in step 1 to stop the port-forward command.
+
+1. Print the log of Pilot and search for `rbac` with the following command:
+
+    > Note: You probably need to first delete and then re-apply your authorization policies so that
+the debug output is generated for these policies.
+
+    {{< text bash >}}
+    $ kubectl logs $(kubectl -n istio-system get pods -l istio=pilot -o jsonpath='{.items[0].metadata.name}') -c discovery -n istio-system | grep rbac
+    {{< /text >}}
+
+1. Check the output and verify:
+
+    * There are no errors.
+    * There is a `"built filter config for ..."` message which means the filter is generated
+      for the target service.
+
+1. For example, you might see something similar to the following:
+
+    {{< text plain >}}
+    2018-07-26T22:25:41.009838Z debug rbac building filter config for {sleep.foo.svc.cluster.local map[app:sleep pod-template-hash:3326367878] map[destination.name:sleep destination.namespace:foo destination.user:default]}
+    2018-07-26T22:25:41.009915Z info  rbac no service role in namespace foo
+    2018-07-26T22:25:41.009957Z info  rbac no service role binding in namespace foo
+    2018-07-26T22:25:41.010000Z debug rbac generated filter config: { }
+    2018-07-26T22:25:41.010114Z info  rbac built filter config for sleep.foo.svc.cluster.local
+    2018-07-26T22:25:41.182400Z debug rbac building filter config for {productpage.default.svc.cluster.local map[pod-template-hash:2600844901 version:v1 app:productpage] map[destination.name:productpage destination.namespace:default destination.user:bookinfo-productpage]}
+    2018-07-26T22:25:41.183131Z debug rbac checking role app2-grpc-viewer
+    2018-07-26T22:25:41.183214Z debug rbac role skipped for no AccessRule matched
+    2018-07-26T22:25:41.183255Z debug rbac checking role productpage-viewer
+    2018-07-26T22:25:41.183281Z debug rbac matched AccessRule[0]
+    2018-07-26T22:25:41.183390Z debug rbac generated filter config: {policies:<key:"productpage-viewer" value:<permissions:<and_rules:<rules:<or_rules:<rules:<header:<name:":method" exact_match:"GET" > > > > > > principals:<and_ids:<ids:<any:true > > > > >  }
+    2018-07-26T22:25:41.184407Z info  rbac built filter config for productpage.default.svc.cluster.local
+    {{< /text >}}
+
+    It means Pilot have generated:
+
+    * An empty config for `sleep.foo.svc.cluster.local` as there is no authorization policies matched
+      and Istio will deny all requests sent to this service by default.
+
+    * An config for `productpage.default.svc.cluster.local` and Istio will allow anyone to access it
```

Comment:
Article, relative pronoun, and markup:
* A configuration file for `productpage.default.svc.cluster.local` and Istio allows anyone to access to the file with the `GET` method.